doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -201,28 +219,56 @@ cells:
     # a  | b  | data  |
     # a0 | b0 | 0     |
     # a0 | b1 | 1     |
-    # a1 | b0 | 2     |
-    # a1 | b1 | 3     |
+    # a0 | b2 | 2     |
+    # a1 | b0 | 3     |
+    # a1 | b1 | 4     |
+    # a1 | b2 | 5     |
 
-  id: 8
+  id: 9
 
 - markdown: |
-    ##### Exporting only data
+    ##### Exporting only data (without headers)
+    
+    To export only data in Excel sheets, you can choose between setting the ``header`` argument of the ``dump`` method to False or avoid calling ``dump``. 
+    
+    This can be usefull as you to export data following a specific format. For example, let's consider you have a 2D array `pop` representing the projection of the population of some selected cities until 2025. Let's assume this array has two axes `city` and `time` respectively but you need to include the associated continent in the output file. What you want is:  
     
-    To export only data in Excel sheets, you can set argument ``header`` of the method ``dump`` to false or not calling ``dump``:
+    |city     |	continent	  | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 | 2022 | 2023 | 2024 | 2025 |
+    | ------- | ------------- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+    |Brussel  |	Europe	      | 0    | 1	| 2	   | 3	  | 4    | 5    | 6    | 7    | 8    | 9    | 10   |
```

Comment:
Nice example!! I would have used fewer years though. Beyond 4 or 5, it doesn't add anything to the example and it makes it unnecessarily big.